README.md

```diff
@@ -98,13 +98,13 @@ We will interact with the following simple contract `kv-store`. In our examples,
 (define-map store ((key (buff 32))) ((value (buff 32))))
 
 (define-public (get-value (key (buff 32)))
-    (match (map-get? store ((key key)))
+    (match (map-get? store {key key})
```

Comment:
This README needs to be refreshed with the new syntax (including the separators).

Reply:
Commit 5b92360 appends decorative colons to the labels of tuple literals in the examples.